src/OrchardCore/OrchardCore.DisplayManagement/ModelBinding/ModelBinderAccessorFilter.cs

```diff
@@ -19,5 +20,23 @@ public void OnActionExecuting(ActionExecutingContext context)
                 modelBinderAccessor.ModelUpdater = new ControllerModelUpdater(controller);
             }
         }
+        public void OnPageHandlerSelected(PageHandlerSelectedContext context)
+        {
+            var page = context.HandlerInstance as PageModel;
+
+            if (page != null)
+            {
+                var modelBinderAccessor = context.HttpContext.RequestServices.GetRequiredService<IUpdateModelAccessor>();
+                modelBinderAccessor.ModelUpdater = new PageModelUpdater(page);
```

Comment:
Is it better to do it here? Just tried in `OnPageHandlerExecuting()`, it also works

I suggest it because the `ControllerModelUpdater` is set in `OnActionExecuting()`